hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/AlertsEngineImpl.java

```diff
@@ -214,11 +214,11 @@ public void reloadTrigger(final String tenantId, final String triggerId) {
         try {
             trigger = definitions.getTrigger(tenantId, triggerId);
         } catch (Exception e) {
-            log.debugf(e.getMessage(), e);
+            log.debug(e.getMessage(), e);
             msgLog.errorDefinitionsService("Trigger", e.getMessage());
         }
         if (null == trigger) {
-            log.debugf("Trigger not found for triggerId [" + triggerId + "], removing from rulebase if it exists");
+            log.debug("Trigger not found for triggerId [" + triggerId + "], removing from rulebase if it exists");
```

Comment:
+1
Yeap, log.debugEnabled() is a good suggestion here, I'm going to review it to avoid unnecessary string manipulation.